src/app/archbishop/archbishop.py

```diff
@@ -97,14 +98,15 @@ def get_player(event, group):
         slack_user_url = config.SLACK_API.format("users.info?user=" + event.user)
         try:
             result = requests.get(slack_user_url, headers={'Authorization': 'Bearer ' + config.BOT_TOKEN})
-            user = result.json().get("user").get("real_name").split()[0]
-            potential_player = [p for p in group.players if p.name == user]
+            user_name = result.json().get("user").get("real_name").split()[0]
```

Comment:
Would you consider this instead?

```
result_json = result.json()
user = result_json.get("user")
user_name = user.get("real_name")
user_id = user.get("id")
```

Reply:
That would certainly make it more readable, I will apply this suggestion